metrictank/src/main/java/com/expedia/metrics/metrictank/MetricKey.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.metrics.metrictank;
+
+import java.util.Objects;
+
+public class MetricKey {
+    private final int orgId;
+    private final String id;
```

Comment:
`id` can be stored as a `byte[]`, it doesn't needed to be convert to a String except in `toString()`. Make sure you switch to `Arrays.equals` in `equals()`